src/board.c

```diff
@@ -28,18 +28,58 @@
 #include "osdvar.h"
 #include <math.h>
 
+
 void vTaskHeartBeat(void *pvParameters);
 void vTask10HZ(void *pvParameters);
 void triggerVideo(void);
 void triggerPanel(void);
 void checkDefaultParam(void);
 
+// Intra-task communication semaphores
+extern xSemaphoreHandle onScreenDisplaySemaphore;
+extern xSemaphoreHandle onMavlinkSemaphore;
+extern xSemaphoreHandle onUAVTalkSemaphore;
```

Comment:
not sure if we need the `UAVTalk` and `Mavlink` one or just use `Telemetry` or something and reuse one, but not really important.
Just something to ponder - if one were to add msp, naza etc it get's somewhat bloaty.